Scripts/script-Ping.yml

```diff
@@ -16,10 +16,10 @@ script: |-
 
       verbose = demisto.get(demisto.args(),'verbose')
       if verbose and verbose.lower() == 'false':
-          demisto.results(res)
+          demisto.results({'ContentsFormat': 'json', 'Type': entryTypes['note'], 'Contents': res, 'EntryContext': { "Ping.Result": res }})
```

Comment:
Use single quote on Ping.Result
I think we have consts for entry format as well

Reply:
Didn't find any const